### DISCUSSION TITLE
Methodology for pressure sensor

### DISCUSSION THREAD

**Discussion by nimeeshrathi** [InteractiveComputerGraphics/SPlisHSPlasH#287] [Q&A] [answered]
Hi Jan ,
Currently I am working on SPHeric test number 2 ( Dam break with obstacle) . Wave formation and splash looks pretty good when we compare to experimental . Main issue arise how I can plot pressure mapping over a obstacle due to fluid  to compare with the experimental result . Any guidance how we can plot pressure ?
![dambreak](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH/assets/93661005/b1f50fbe-e29e-4152-9efd-350890f58ae8)

**Comment by janbender** [comment, marked as answer]
There is a Python example which demonstrates how to plot some values. Maybe you can extend this example for your scene.
